package.json

```diff
@@ -34,12 +34,11 @@
     "example:syncValidation": "form=syncValidation npm run example",
     "example:wizard": "form=wizard npm run example",
     "format": "prettier-eslint \"+(src|examples)/**/*.+(js|js.flow)\" --ignore \"**/bundle.js\" --write --no-semi --single-quote --trailing-comma=none",
-    "prepublish": "npm run test && npm run test:flow && npm run clean && npm run build",
+    "prepublish": "npm run lint && npm run test:cov && npm run test:flow && npm run clean && npm run build",
     "test": "cross-env NODE_ENV=test jest --runInBand",
     "test:flow": "flow check",
     "test:watch": "npm test -- --watch",
-    "test:cov": "npm run test -- --coverage",
-    "test:codecov": "cat ./coverage/lcov.info | ./node_modules/codecov.io/bin/codecov.io.js"
```

Comment:
`test:codecov` is dead code

Reply:
Codecov no longer requires that `codecov.io.js` to be executed?

Reply:
It does. Let me make a PR now that fixes this (and ensure it works together with Jest). 

Not sure the current coverage of 100% is correct either.

Reply:
@erikras https://github.com/erikras/redux-form/pull/3475